openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -442,7 +444,7 @@ The technology described in this specification was made available from contribut
 
    -04
 
-   * ...
+   * Define key resolution mechanisms for all elements
```

Comment:
I think this underplays "x509_san_dns & verifier_attestations client id prefixes are no longer permitted, x509_hash must be used"

Reply:
```suggestion
   * x509_san_dns & verifier_attestations client id prefixes are no longer permitted, x509_hash must be used
```